extensions/lark/package.json

```diff
@@ -0,0 +1,91 @@
+{
+  "name": "lark",
+  "title": "Lark",
+  "version": "0.0.1",
+  "description": "Search docs, .etc.",
```

Comment:
Please give more comprehensive description. Something like "Quickly search documents from Lark or Feishu" (I'm not sure what's the difference between the two, but you get the idea.